servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -279,6 +490,77 @@ public Builder ewmaHalfLife(final Duration ewmaHalfLife) {
+         * @return {@code this}
+         */
+        public Builder ewmaErrorPenalty(final long ewmaErrorPenalty) {
+            this.ewmaErrorPenalty = ensureNonNegative(ewmaErrorPenalty, "ewmaErrorPenalty");
+            return this;
+        }
+
+        /**
+         * Configure an interval for re-subscribing to the original events stream in case all existing hosts become
+         * unhealthy.
+         * <p>
+         * In situations when there is a latency between {@link ServiceDiscoverer} propagating the updated state and all
+         * known hosts become unhealthy, which could happen due to intermediate caching layers, re-subscribing to the
+         * events stream can help to exit from a dead state.
+         * <p>
+         * Note: setting the interval to {@code Duration.ofNanos(Long.MAX_VALUE)} will effectively disable health check
+         * resubscribes.
+         *
+         * @param interval interval at which re-subscribes will be scheduled.
+         * @param jitter the amount of jitter to apply to each re-subscribe {@code interval}.
+         * @return {@code this}.
+         */
+        public Builder serviceDiscoveryResubscribeInterval(Duration interval, Duration jitter) {
+            validateHealthCheckIntervals(interval, jitter);
+            this.serviceDiscoveryResubscribeInterval = interval;
+            this.serviceDiscoveryResubscribeJitter = jitter;
+            return this;
+        }
+
+        /**
+         * Configure a threshold for consecutive connection failures to a host. When the {@link LoadBalancer}
+         * consecutively fails to open connections in the amount greater or equal to the specified value,
+         * the host will be marked as unhealthy and connection establishment will take place in the background
+         * repeatedly on the {@link #interval()} (with jitter {@link #intervalJitter()}) until a connection is
+         * established. During that time, the host will not take part in load balancing selection.
+         * <p>
+         * Use a negative value of the argument to disable health checking.
+         *
+         * @param failedConnectionsThreshold number of consecutive connection failures to consider a host unhealthy and
+         *                                   eligible for background health checking. Use negative value to disable the
+         *                                   health checking mechanism.
+         * @return {@code this}.
+         */
+        Builder failedConnectionsThreshold(int failedConnectionsThreshold) {
+            this.failedConnectionsThreshold = failedConnectionsThreshold;
+            if (failedConnectionsThreshold == 0) {
+                throw new IllegalArgumentException("Not valid value: 0");
+            }
+            return this;
+        }
+
         /**
          * Set the threshold for consecutive failures before a host is ejected.
          * Defaults to 5.
```

Comment:
Here and in all other cases where you document default values, consider using `{@value DEFAULT_CONSECUTIVE_5XX}` to make sure javadoc does not become outdated if we change the default value later.

Reply:
TIL